pkg/shell/shell.css

```diff
@@ -351,6 +351,25 @@ ul.available-interfaces-group {
     cursor: pointer;
 }
 
+#account-authorized-keys-list td:first-child {
+    max-width: 100px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+#account-authorized-keys-list td.fingerprint {
+    max-width: 200px;
```

Comment:
Is there a reason for this width? I'd expect a min-width maybe. You could give the column for the buttons a fixed width and calculate the width of the fingerprint column accordingly (see journal).